### PR TITLE
Add Italian translation

### PR DESCRIPTION
**Pull request by quazar-omega**
Hello!  
I'd like to add Italian translations to this project, are there other files I need to touch to finalize the PR?

### CODE REVIEW

**Comment by vnepogodin** [comment]
Hey @quazar-omega

Many thanks for contribution!

All in all looks good.

You need to also add "<item id="it">Italian</item>" here after "German" lang(https://github.com/CachyOS/CachyOS-Welcome/blob/develop/ui/cachyos-hello.glade#L287).

Also if you want to translate "pages":
- create folder "pages/it"
- copy files from "pages/en" to "pages/it"
- then translate

"involved", "readme" are less likely to change anytime
"release" changes with every release

Best regards

File: i18n/it/cachyos_hello.ftl
Comment (NextWork123) [inline i18n/it/cachyos_hello.ftl]: much probably it can be more appropriate to write like that:

```
Grazie per unirti alla nostra communità!

Noi, gli sviluppatori di CachyOS, speriamo che tu trovi l'utilizzo di CachyOS altrettanto piacevole quanto lo troviamo noi nello svilupparlo. I link sottostanti ti aiuteranno a orientarti nel tuo nuovo sistema operativo. Goditi questa esperienza e non esitare a inviarci i tuoi feedback
```

File: i18n/it/cachyos_hello.ftl
Comment (NextWork123) [inline i18n/it/cachyos_hello.ftl]: please replace `Installa App` with `Installa Applicazioni` 

and instead of `App/Modifiche` replace with `Applicazioni/Personalizzazioni`

File: i18n/it/cachyos_hello.ftl
Comment (NextWork123) [inline i18n/it/cachyos_hello.ftl]: instead to use `dona` is more appropriate to use `donazioni`

File: i18n/it/cachyos_hello.ftl
Comment (NextWork123) [inline i18n/it/cachyos_hello.ftl]: i think it can be more appropriate to use `Note sulla versione`

**Review by NextWork123 — CHANGES_REQUESTED** [review]

**Comment by quazar-omega** [comment]
@vnepogodin Glad to be of help!  
I'll look into the other files as soon as possible :)

File: i18n/it/cachyos_hello.ftl
Comment (quazar-omega) [reply i18n/it/cachyos_hello.ftl]: I'm trying to stay faithful to the source material as much as possible, since the same abbreviation for "Application" exists de facto in Italian too, I kept that in.  
I like the change `Modifiche -> Personalizzazioni` though, I can do that. Of course, if you'd still like me to change "App" I'll do that too for the sake of consistency.

File: i18n/it/cachyos_hello.ftl
Comment (quazar-omega) [reply i18n/it/cachyos_hello.ftl]: If I do that I'll have to change the source string too, otherwise it'd be inaccurate

File: i18n/it/cachyos_hello.ftl
Comment (NextWork123) [reply i18n/it/cachyos_hello.ftl]: I prefer you change to `Applicazioni` is more easy for people to understand.

File: i18n/it/cachyos_hello.ftl
Comment (quazar-omega) [reply i18n/it/cachyos_hello.ftl]: Hmm, I like most of the second part a lot, how about this:
```ftl
welcome-body =
    Grazie per esserti unito alla nostra comunità! # "Grazie per unirti" sounds very unnatural to me

    Noi, gli sviluppatori di CachyOS, speriamo che trovi l'utilizzo di CachyOS altrettanto piacevole quanto lo troviamo noi nello svilupparlo. I collegamenti sottostanti ti aiuteranno a orientarti nel tuo nuovo sistema operativo. Per cui goditi l'esperienza e non esitare a inviarci la tua opinione. # I'd like to avoid anglicisms where possible
```

File: i18n/it/cachyos_hello.ftl
Comment (NextWork123) [reply i18n/it/cachyos_hello.ftl]: it gonna be inaccurate anyway because you translate with `dona` but if you translate in italian `donate` it gonna be `donare`. So i don't think you need to change source string for use `donazioni`.

File: i18n/it/cachyos_hello.ftl
Comment (quazar-omega) [reply i18n/it/cachyos_hello.ftl]: Ok, I've seen both being used more or less, I can pick yours up!

File: i18n/it/cachyos_hello.ftl
Comment (NextWork123) [reply i18n/it/cachyos_hello.ftl]: yeah is perfect that!

File: i18n/it/cachyos_hello.ftl
Comment (NextWork123) [reply i18n/it/cachyos_hello.ftl]: is more accurated only that. anyway thanks for fixing all things!

File: i18n/it/cachyos_hello.ftl
Comment (quazar-omega) [reply i18n/it/cachyos_hello.ftl]: Hmm, I think you misunderstood my translation there, since most buttons represent an action they are in imperative form, both in English and Italian, so "donate" maps to "dona", if it were "donazioni" on the Italian side, I'd have "donations" in English

Comment (NextWork123) [reply i18n/it/cachyos_hello.ftl]: Hmm, i understand but i don't like so much `dona` is a bit weird to read, what about instead to use `dona` we can replace with `supportaci` is more used to all companies and i like that?

Comment (quazar-omega) [reply i18n/it/cachyos_hello.ftl]: Good point!  
Perhaps that should reflect on the English one too for consistency? I'll leave that up to you though, I think "Supportaci" still works on its own so I'll change it either way

Comment (NextWork123) [reply i18n/it/cachyos_hello.ftl]: Nah, leave that don't reflect all translation very closely to english. Anyway yeah please replace with `Supportaci` looks more professional

**Review by NextWork123 — APPROVED** [review]
Thanks for fixing, now is perfect!

Waiting you gonna finish all the translation at 100%

**Comment by quazar-omega** [comment]
You got it!  
I'll see when I can translate the rest of the files

**Comment by quazar-omega** [comment]
Oops, thanks for that 😅

**Comment by NextWork123** [comment]
> Oops, thanks for that 😅

Np, i will stay review all xD 

because i'm the only member from italy xD

**Comment by quazar-omega** [comment]
Well, hey, ci si aiuta fra connazionali! ;)

**Comment by NextWork123** [comment]
> Well, hey, ci si aiuta fra connazionali! ;)

Ohoh, perfetto!

**Comment by NextWork123** [comment]
@vnepogodin thanks for reviewed my commit and fixed again <3

**Comment by NextWork123** [comment]
@quazar-omega Hello, wanted to know if is all good on my commit if you have some suggestion or other.

**Comment by quazar-omega** [comment]
Hey! Sorry for the late response, was busy these last few days.

I'll be looking at them and proofread between today and tomorrow.  
First file is looking great with some _really_ minor nitpicky corrections, I'll see the rest tomorrow

**Comment by NextWork123** [comment]
> Hey! Sorry for the late response, was busy these last few days.
> 
> I'll be looking at them and proofread between today and tomorrow. First file is looking great with some _really_ minor nitpicky corrections, I'll see the rest tomorrow

Ok!, i will wait for your response

**Review by NextWork123 — APPROVED** [review]
Thanks for the correction seem all fine!

**Comment by quazar-omega** [comment]
Yes the translation was on point, good job!

What about the dashes in the list changed to space indentation btw?

**Comment by NextWork123** [comment]
> Yes the translation was on point, good job!
> 
> What about the dashes in the list changed to space indentation btw?

Yeah seem all good you need to change other? if is all good too for you i gonna merge

**Comment by quazar-omega** [comment]
Yes, you can go ahead!

**Pull request merged**
